src/manifest.ts

```diff
@@ -676,15 +681,17 @@ export class Manifest {
       );
       return;
     }
-    const packagesForReleasers = await this.getPackagesToRelease(
-      // use the lastMergedPR.sha as a Commit: lastMergedPR.files will inform
-      // getPackagesToRelease() what packages had changes (i.e. at least one
-      // file under their path changed in the lastMergedPR such as
-      // "packages/mypkg/package.json"). These are exactly the packages we want
-      // to create releases/tags for.
-      [{sha: lastMergedPR.sha, message: '', files: lastMergedPR.files}],
-      lastMergedPR.sha
-    );
+    const packagesForReleasers = (
+      await this.getPackagesToRelease(
+        // use the lastMergedPR.sha as a Commit: lastMergedPR.files will inform
+        // getPackagesToRelease() what packages had changes (i.e. at least one
+        // file under their path changed in the lastMergedPR such as
+        // "packages/mypkg/package.json"). These are exactly the packages we want
+        // to create releases/tags for.
+        [{sha: lastMergedPR.sha, message: '', files: lastMergedPR.files}],
+        lastMergedPR.sha
+      )
+    ).filter(pkg => !pkg.config.skipGithubRelease);
```

Comment:
What do you think about a slightly different approach: rather than just exclude them what if we checked this flag further down and did a `this.gh.commentOnIssue("`:robot: ${pkgName} not configured for release :no_entry_sign:`"). Easier for after-the-fact troubleshooting (rather than looking at git history on `release-please-config.json` and tying the timeline to release creation runs)

I also think it handles this edge case better: all packages are marked as `skip-github-release` - that way we still "release" the PR (remove the `autorelease: pending` label and add the `autorelease: tagged` label). That doesn't really affect functionality (we only ever look at the `lastMergedPR`) but for historical viewing/reporting I think it makes more sense to see a "released" PR and it just so happens that no package was actually released.

Reply:
Yp totally agreed.
I will update it & I will try adding some tests